shared/parser.go

```diff
@@ -351,7 +351,7 @@ func Parse(argList []string) ParserResult {
 		"-fopenmp":       {0, pr.compileLinkUnaryCallback},
 
 		"-Wl,-dead_strip": {0, pr.warningLinkUnaryCallback},
-		"-dead_strip":     {0, pr.warningLinkUnaryCallback}, //iam: tor does this. We lose the bitcode :-(
+		"-dead_strip":     {0, pr.warningLinkUnaryCallback},  //iam: tor does this. We lose the bitcode :-(
```

Comment:
I think this line shouldn't have been changed?

Reply:
I think we had a similar problem last time, and there must be `go fmt` acting somehow... I'll fix it.

Reply:
Fix in [0412d29](https://github.com/capablevms/gllvm/pull/5/commits/0412d29cd92d255ae7ffb1e23fa4f0f3f8d4f04b).